hello_gadget/mix.exs

```diff
@@ -0,0 +1,78 @@
+defmodule HelloGadget.Mixfile do
+  use Mix.Project
+
+  @target System.get_env("MIX_TARGET") || "host"
+
+  Mix.shell.info([:green, """
+  Mix environment
+    MIX_TARGET:   #{@target}
+    MIX_ENV:      #{Mix.env}
+  """, :reset])
+
+  def project do
+    [app: :hello_gadget,
+     version: "0.3.0",
```

Comment:
It doesn't really matter, but this is the first version of this example, so maybe make it `0.1.0`